jenkins/scripts/integration_test.sh

```diff
@@ -19,27 +19,38 @@ CI_DIR="$(dirname "$(readlink -f "${0}")")"
 # shellcheck disable=SC1090
 source "${CI_DIR}/utils.sh"
 
-VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
-TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
-TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
-TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-100GB}"
-
+DISTRIBUTION="${DISTRIBUTION:-ubuntu}"
+if [ "${DISTRIBUTION}" == "ubuntu" ]
+then
+  IMAGE_NAME="${CI_METAL3_IMAGE}"
+  IMAGE_OS="Ubuntu"
+else
+  IMAGE_NAME="${CI_METAL3_CENTOS_IMAGE}"
+  IMAGE_OS="Centos"
```

Comment:
Maybe we should change `TARGET_NODE_OS` to `"Centos"` in centos related JJB files? so that there is no need to set it explicitly here?

Reply:
This is for testing for now, I don't want to break the existing tests as I am not sure that the replacement works fine.